book/src/Setup.md

```diff
@@ -171,7 +171,7 @@ To configure your driver using the registrar sidecar, you can configure the cont
         fieldRef:
           apiVersion: v1
           fieldPath: spec.nodeName
-    image: quay.io/k8scsi/driver-registrar:v0.2.0
+    image: quay.io/k8scsi/driver-registrar:v1.0.0
```

Comment:
Move this to 1.0.1
1.0.0 has a bug that will prevent it from working (it advertises the wrong supported version).

Reply:
And replace driver-registrar with node-driver-registrar.